server/src/configs/sequelize.js

```diff
@@ -0,0 +1,36 @@
+require("dotenv").config();
+const { Sequelize } = require("sequelize");
+
+try {
```

Comment:
We should not need to wrap the sequelize creation in a try-catch anymore now that Github actions has been updated

Reply:
I removed them and updated the runner file to ensure that all environment variables are available for the connection to be established properly😀